### PR TITLE
Update butterfly for v2 specs actors

### PR DESCRIPTION
**Pull request by travisperson**
Constructed via
```
$ git checkout -b update/ntwk-butterfly be9d23b
$ git merge ntwk-butterfly
```

### CODE REVIEW

File: build/params_testnet.go [hunk 1]
```diff
@@ -20,22 +20,21 @@ const BreezeGasTampingDuration = 120
 
 const UpgradeSmokeHeight = -1
 
-const UpgradeIgnitionHeight = 94000
+const UpgradeIgnitionHeight = -1
```
Comment (travisperson) [inline build/params_testnet.go:23]: This might actually need to be set to something.

Comment (travisperson) [reply build/params_testnet.go]: I've pushed another commit which will set this to `23950`.

**Comment by arajasek** [comment]
Sorry, totally forgot about upgrade epochs! It should be fine to set -1 for all epochs except `UpgradeActorsV2` which we can set to, say, 500?

Also Idk if some of the tests broke over the merge...Might be easier to just hard reset `ntwk-butterfly` to `be9d23b`, and then merge a "setup network and epochs" commit on top? `be9d23b` should already have most of latest `master` in it.

File: build/params_testnet.go (hunk 1 above)
Comment (arajasek) [reply build/params_testnet.go]: It _should_ be fine to set it to -1

**Comment by arajasek** [comment]
Not being merged

**Pull request closed without merging**